### DISCUSSION TITLE
Welcome to Telegram-Trilateration Discussions!

### DISCUSSION THREAD

**Discussion by jkctech** [jkctech/Telegram-Trilateration#1] [Announcements]
## 👋 Hey there

If you have any questions or things you would like to say, please leave them here :)

**Comment by Acern0** [comment]
Hello, I was trying to test your proof of concept for this but I have encountered a problem, the resolution of the screen for the OCR part.
Can you please tell me what resolution did u use for both the PC screen and the Nox emulator? Because if i want to modify the pixel location in settings.py, by my own resolution, it's gonna take a lot of time and I might mess up some of the other settings you have done.
